package.json

```diff
@@ -1,11 +1,13 @@
 {
   "name": "electron-reload",
-  "version": "1.5.0",
+  "version": "1.6.1",
```

Comment:
I don't think this is correct, most specifically having a patch version without no hot fixes/patches is not correct. Although the API stays the same, I would have imagined that version `2.0.0` would be more appropriate to avoid any surprises for the users.

Reply:
fixed: check latest. Bumped version to `2.0.0`